scripts/suse/dpk/README.md

```diff
@@ -0,0 +1,9 @@
+## What it is
+
+Debian base package to notify instalation of new packages outside the control of uyuni/suse manager
```

Comment:
Strictly speaking the script will collect dpkg operation outside of Salt, not Uyuni or SUSE Manager. 
```suggestion
Debian base package to notify installation of new packages outside the control of Uyuni/SUSE Manager
```